_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -38941,7 +38943,9 @@
 /obj/structure/cable/yellow{
 	icon_state = "0-4"
 	},
-/obj/machinery/power/smes/engineering,
+/obj/machinery/power/smes/engineering{
+	charge = 3e+006
```

Comment:
Just set the default charge in the code, rather then changing it in all maps. Code is in /code/modules/power/smes.dm

Reply:
gotcha, changed the smes/engineering default charge instead